From 65cd8b2bd0e4cb961836d2fe55672fd9d7d4a8b4 Mon Sep 17 00:00:00 2001
From: John Audia <therealgraysky@proton.me>
Date: Wed, 21 Jun 2023 12:17:06 -0400
Subject: [PATCH] kernel: bump 5.15 to 5.15.118

All patches automatically rebased.

Build system: x86_64
Build-tested: ramips/tplink_archer-a6-v3, filogic/xiaomi_redmi-router-ax6000-ubootmod
Run-tested: ramips/tplink_archer-a6-v3, filogic/xiaomi_redmi-router-ax6000-ubootmod

Signed-off-by: John Audia <therealgraysky@proton.me>
---
 include/kernel-5.15                                         | 4 ++--
 target/linux/ath25/patches-5.15/107-ar5312_gpio.patch       | 2 +-
 .../patches-5.15/0032-MIPS-ath79-sanitize-symbols.patch     | 2 +-
 .../ath79/patches-5.15/910-unaligned_access_hacks.patch     | 2 +-
 .../322-MIPS-BCM63XX-switch-to-IRQ_DOMAIN.patch             | 2 +-
 .../patches-5.15/202-mips-bmips-tweak-Kconfig-options.patch | 2 +-
 .../bmips/patches-5.15/600-mips-bmips-add-pci-support.patch | 2 +-
 .../600-v5.18-page_pool-Add-allocation-stats.patch          | 2 +-
 ...707-v6.3-net-pcs-add-driver-for-MediaTek-SGMII-PCS.patch | 2 +-
 ...0011-net-dsa-mt7530-introduce-separate-MDIO-driver.patch | 2 +-
 ...t-dsa-mt7530-introduce-driver-for-MT7988-built-in-.patch | 2 +-
 ...y-Add-driver-for-Motorcomm-yt8521-gigabit-ethernet.patch | 2 +-
 ...-0009-nvmem-Add-driver-for-OCOTP-in-Sunplus-SP7021.patch | 2 +-
 .../806-v6.0-0001-nvmem-microchip-otpc-add-support.patch    | 2 +-
 ...-v5.17-net-qmi_wwan-add-ZTE-MF286D-modem-19d2-1485.patch | 2 +-
 target/linux/generic/hack-5.15/251-kconfig.patch            | 2 +-
 target/linux/generic/hack-5.15/280-rfkill-stubs.patch       | 2 +-
 .../generic/hack-5.15/721-net-add-packet-mangeling.patch    | 6 +++---
 .../hack-5.15/780-usb-net-MeigLink_modem_support.patch      | 2 +-
 .../generic/pending-5.15/300-mips_expose_boot_raw.patch     | 4 ++--
 .../680-NET-skip-GRO-for-foreign-MAC-addresses.patch        | 2 +-
 ...t-core-add-optional-threading-for-backlog-processi.patch | 6 +++---
 target/linux/lantiq/patches-5.15/0152-lantiq-VPE.patch      | 2 +-
 .../patches-5.15/701-staging-add-fsl_ppfe-driver.patch      | 2 +-
 ...t-phy-add-driver-for-MediaTek-SoC-built-in-GE-PHYs.patch | 2 +-
 ...INTAINERS-Add-an-entry-for-the-IEI-WT61P803-PUZZLE.patch | 2 +-
 .../octeon/patches-5.15/100-mips_image_cmdline_hack.patch   | 2 +-
 .../314-MIPS-add-bootargs-override-property.patch           | 4 ++--
 .../patches-5.15/315-owrt-hack-fix-mt7688-cache-issue.patch | 4 ++--
 .../008-5.17-watchdog-add-realtek-otto-watchdog-timer.patch | 2 +-
 .../patches-5.15/300-mips-add-rtl838x-platform.patch        | 2 +-
 31 files changed, 39 insertions(+), 39 deletions(-)

diff --git a/include/kernel-5.15 b/include/kernel-5.15
index fc637450b53df..aa4541d277a71 100644
--- a/include/kernel-5.15
+++ b/include/kernel-5.15
@@ -1,2 +1,2 @@
-LINUX_VERSION-5.15 = .117
-LINUX_KERNEL_HASH-5.15.117 = a9a8331f3c0e004b40a41f30d688b53bf85405597ac41c2c89be7f4fbcf7239f
+LINUX_VERSION-5.15 = .118
+LINUX_KERNEL_HASH-5.15.118 = 4e6bf4dadb04d5d11d1d4cc37c0eabcf33bc333b7dd3dc2143c3099a823eb5b3
diff --git a/target/linux/ath25/patches-5.15/107-ar5312_gpio.patch b/target/linux/ath25/patches-5.15/107-ar5312_gpio.patch
index 4ad1d40facd2f..e6e1290d35a4b 100644
--- a/target/linux/ath25/patches-5.15/107-ar5312_gpio.patch
+++ b/target/linux/ath25/patches-5.15/107-ar5312_gpio.patch
@@ -202,7 +202,7 @@
 +subsys_initcall(ar5312_gpio_init);
 --- a/arch/mips/Kconfig
 +++ b/arch/mips/Kconfig
-@@ -230,6 +230,7 @@ config ATH25
+@@ -231,6 +231,7 @@ config ATH25
  	select CEVT_R4K
  	select CSRC_R4K
  	select DMA_NONCOHERENT
diff --git a/target/linux/ath79/patches-5.15/0032-MIPS-ath79-sanitize-symbols.patch b/target/linux/ath79/patches-5.15/0032-MIPS-ath79-sanitize-symbols.patch
index 9bce3a2633305..9d014434ecb33 100644
--- a/target/linux/ath79/patches-5.15/0032-MIPS-ath79-sanitize-symbols.patch
+++ b/target/linux/ath79/patches-5.15/0032-MIPS-ath79-sanitize-symbols.patch
@@ -15,7 +15,7 @@ Signed-off-by: John Crispin <john@phrozen.org>
 
 --- a/arch/mips/Kconfig
 +++ b/arch/mips/Kconfig
-@@ -256,6 +256,8 @@ config ATH79
+@@ -257,6 +257,8 @@ config ATH79
  	select SYS_SUPPORTS_BIG_ENDIAN
  	select SYS_SUPPORTS_MIPS16
  	select SYS_SUPPORTS_ZBOOT_UART_PROM
diff --git a/target/linux/ath79/patches-5.15/910-unaligned_access_hacks.patch b/target/linux/ath79/patches-5.15/910-unaligned_access_hacks.patch
index c2b646643e038..ad685fb13b81f 100644
--- a/target/linux/ath79/patches-5.15/910-unaligned_access_hacks.patch
+++ b/target/linux/ath79/patches-5.15/910-unaligned_access_hacks.patch
@@ -784,7 +784,7 @@ SVN-Revision: 35130
  
 --- a/include/net/neighbour.h
 +++ b/include/net/neighbour.h
-@@ -275,8 +275,10 @@ static inline bool neigh_key_eq128(const
+@@ -270,8 +270,10 @@ static inline bool neigh_key_eq128(const
  	const u32 *n32 = (const u32 *)n->primary_key;
  	const u32 *p32 = pkey;
  
diff --git a/target/linux/bcm63xx/patches-5.15/322-MIPS-BCM63XX-switch-to-IRQ_DOMAIN.patch b/target/linux/bcm63xx/patches-5.15/322-MIPS-BCM63XX-switch-to-IRQ_DOMAIN.patch
index a848165ac5f51..aff3f4a1a0e4b 100644
--- a/target/linux/bcm63xx/patches-5.15/322-MIPS-BCM63XX-switch-to-IRQ_DOMAIN.patch
+++ b/target/linux/bcm63xx/patches-5.15/322-MIPS-BCM63XX-switch-to-IRQ_DOMAIN.patch
@@ -14,7 +14,7 @@ Signed-off-by: Jonas Gorski <jogo@openwrt.org>
 
 --- a/arch/mips/Kconfig
 +++ b/arch/mips/Kconfig
-@@ -331,6 +331,9 @@ config BCM63XX
+@@ -332,6 +332,9 @@ config BCM63XX
  	select SYNC_R4K
  	select DMA_NONCOHERENT
  	select IRQ_MIPS_CPU
diff --git a/target/linux/bmips/patches-5.15/202-mips-bmips-tweak-Kconfig-options.patch b/target/linux/bmips/patches-5.15/202-mips-bmips-tweak-Kconfig-options.patch
index c39df850a0344..eb3f1c1d5cecb 100644
--- a/target/linux/bmips/patches-5.15/202-mips-bmips-tweak-Kconfig-options.patch
+++ b/target/linux/bmips/patches-5.15/202-mips-bmips-tweak-Kconfig-options.patch
@@ -13,7 +13,7 @@ Signed-off-by: Álvaro Fernández Rojas <noltari@gmail.com>
 
 --- a/arch/mips/Kconfig
 +++ b/arch/mips/Kconfig
-@@ -274,25 +274,20 @@ config BMIPS_GENERIC
+@@ -275,25 +275,20 @@ config BMIPS_GENERIC
  	select SYNC_R4K
  	select COMMON_CLK
  	select BCM6345_L1_IRQ
diff --git a/target/linux/bmips/patches-5.15/600-mips-bmips-add-pci-support.patch b/target/linux/bmips/patches-5.15/600-mips-bmips-add-pci-support.patch
index 36b09105b7272..e2ca75a3930e7 100644
--- a/target/linux/bmips/patches-5.15/600-mips-bmips-add-pci-support.patch
+++ b/target/linux/bmips/patches-5.15/600-mips-bmips-add-pci-support.patch
@@ -14,7 +14,7 @@ Signed-off-by: Álvaro Fernández Rojas <noltari@gmail.com>
 
 --- a/arch/mips/Kconfig
 +++ b/arch/mips/Kconfig
-@@ -274,6 +274,7 @@ config BMIPS_GENERIC
+@@ -275,6 +275,7 @@ config BMIPS_GENERIC
  	select SYNC_R4K
  	select COMMON_CLK
  	select BCM6345_L1_IRQ
diff --git a/target/linux/generic/backport-5.15/600-v5.18-page_pool-Add-allocation-stats.patch b/target/linux/generic/backport-5.15/600-v5.18-page_pool-Add-allocation-stats.patch
index f4c5b9afee4dd..dca1062be9c95 100644
--- a/target/linux/generic/backport-5.15/600-v5.18-page_pool-Add-allocation-stats.patch
+++ b/target/linux/generic/backport-5.15/600-v5.18-page_pool-Add-allocation-stats.patch
@@ -65,7 +65,7 @@ Signed-off-by: David S. Miller <davem@davemloft.net>
  struct page *page_pool_alloc_pages(struct page_pool *pool, gfp_t gfp);
 --- a/net/Kconfig
 +++ b/net/Kconfig
-@@ -434,6 +434,19 @@ config NET_DEVLINK
+@@ -432,6 +432,19 @@ config NET_DEVLINK
  config PAGE_POOL
  	bool
  
diff --git a/target/linux/generic/backport-5.15/707-v6.3-net-pcs-add-driver-for-MediaTek-SGMII-PCS.patch b/target/linux/generic/backport-5.15/707-v6.3-net-pcs-add-driver-for-MediaTek-SGMII-PCS.patch
index 9743a3f2062fb..3668a821ef758 100644
--- a/target/linux/generic/backport-5.15/707-v6.3-net-pcs-add-driver-for-MediaTek-SGMII-PCS.patch
+++ b/target/linux/generic/backport-5.15/707-v6.3-net-pcs-add-driver-for-MediaTek-SGMII-PCS.patch
@@ -32,7 +32,7 @@ Signed-off-by: Jakub Kicinski <kuba@kernel.org>
 
 --- a/MAINTAINERS
 +++ b/MAINTAINERS
-@@ -11789,6 +11789,14 @@ L:	netdev@vger.kernel.org
+@@ -11782,6 +11782,14 @@ L:	netdev@vger.kernel.org
  S:	Maintained
  F:	drivers/net/ethernet/mediatek/
  
diff --git a/target/linux/generic/backport-5.15/790-v6.4-0011-net-dsa-mt7530-introduce-separate-MDIO-driver.patch b/target/linux/generic/backport-5.15/790-v6.4-0011-net-dsa-mt7530-introduce-separate-MDIO-driver.patch
index 51346d0620d3a..d5ba5dbf66260 100644
--- a/target/linux/generic/backport-5.15/790-v6.4-0011-net-dsa-mt7530-introduce-separate-MDIO-driver.patch
+++ b/target/linux/generic/backport-5.15/790-v6.4-0011-net-dsa-mt7530-introduce-separate-MDIO-driver.patch
@@ -25,7 +25,7 @@ Signed-off-by: David S. Miller <davem@davemloft.net>
 
 --- a/MAINTAINERS
 +++ b/MAINTAINERS
-@@ -11901,6 +11901,7 @@ M:	Landen Chao <Landen.Chao@mediatek.com
+@@ -11894,6 +11894,7 @@ M:	Landen Chao <Landen.Chao@mediatek.com
  M:	DENG Qingfang <dqfext@gmail.com>
  L:	netdev@vger.kernel.org
  S:	Maintained
diff --git a/target/linux/generic/backport-5.15/790-v6.4-0013-net-dsa-mt7530-introduce-driver-for-MT7988-built-in-.patch b/target/linux/generic/backport-5.15/790-v6.4-0013-net-dsa-mt7530-introduce-driver-for-MT7988-built-in-.patch
index 00f4b25c44e1d..b748fb5f8f19d 100644
--- a/target/linux/generic/backport-5.15/790-v6.4-0013-net-dsa-mt7530-introduce-driver-for-MT7988-built-in-.patch
+++ b/target/linux/generic/backport-5.15/790-v6.4-0013-net-dsa-mt7530-introduce-driver-for-MT7988-built-in-.patch
@@ -28,7 +28,7 @@ Signed-off-by: David S. Miller <davem@davemloft.net>
 
 --- a/MAINTAINERS
 +++ b/MAINTAINERS
-@@ -11899,9 +11899,11 @@ MEDIATEK SWITCH DRIVER
+@@ -11892,9 +11892,11 @@ MEDIATEK SWITCH DRIVER
  M:	Sean Wang <sean.wang@mediatek.com>
  M:	Landen Chao <Landen.Chao@mediatek.com>
  M:	DENG Qingfang <dqfext@gmail.com>
diff --git a/target/linux/generic/backport-5.15/791-v6.2-01-net-phy-Add-driver-for-Motorcomm-yt8521-gigabit-ethernet.patch b/target/linux/generic/backport-5.15/791-v6.2-01-net-phy-Add-driver-for-Motorcomm-yt8521-gigabit-ethernet.patch
index e2a0e25a7941f..eb51b410b5444 100644
--- a/target/linux/generic/backport-5.15/791-v6.2-01-net-phy-Add-driver-for-Motorcomm-yt8521-gigabit-ethernet.patch
+++ b/target/linux/generic/backport-5.15/791-v6.2-01-net-phy-Add-driver-for-Motorcomm-yt8521-gigabit-ethernet.patch
@@ -21,7 +21,7 @@ Signed-off-by: David S. Miller <davem@davemloft.net>
 
 --- a/MAINTAINERS
 +++ b/MAINTAINERS
-@@ -12700,6 +12700,7 @@ F:	include/uapi/linux/meye.h
+@@ -12693,6 +12693,7 @@ F:	include/uapi/linux/meye.h
  
  MOTORCOMM PHY DRIVER
  M:	Peter Geis <pgwipeout@gmail.com>
diff --git a/target/linux/generic/backport-5.15/804-v5.18-0009-nvmem-Add-driver-for-OCOTP-in-Sunplus-SP7021.patch b/target/linux/generic/backport-5.15/804-v5.18-0009-nvmem-Add-driver-for-OCOTP-in-Sunplus-SP7021.patch
index ae177e0cb036e..6ad9257d60e4e 100644
--- a/target/linux/generic/backport-5.15/804-v5.18-0009-nvmem-Add-driver-for-OCOTP-in-Sunplus-SP7021.patch
+++ b/target/linux/generic/backport-5.15/804-v5.18-0009-nvmem-Add-driver-for-OCOTP-in-Sunplus-SP7021.patch
@@ -19,7 +19,7 @@ Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
 
 --- a/MAINTAINERS
 +++ b/MAINTAINERS
-@@ -17965,6 +17965,11 @@ L:	netdev@vger.kernel.org
+@@ -17958,6 +17958,11 @@ L:	netdev@vger.kernel.org
  S:	Maintained
  F:	drivers/net/ethernet/dlink/sundance.c
  
diff --git a/target/linux/generic/backport-5.15/806-v6.0-0001-nvmem-microchip-otpc-add-support.patch b/target/linux/generic/backport-5.15/806-v6.0-0001-nvmem-microchip-otpc-add-support.patch
index ad0460d4f36fd..f7ea61626c506 100644
--- a/target/linux/generic/backport-5.15/806-v6.0-0001-nvmem-microchip-otpc-add-support.patch
+++ b/target/linux/generic/backport-5.15/806-v6.0-0001-nvmem-microchip-otpc-add-support.patch
@@ -57,7 +57,7 @@ Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
 
 --- a/MAINTAINERS
 +++ b/MAINTAINERS
-@@ -12364,6 +12364,14 @@ S:	Supported
+@@ -12357,6 +12357,14 @@ S:	Supported
  F:	Documentation/devicetree/bindings/mtd/atmel-nand.txt
  F:	drivers/mtd/nand/raw/atmel/*
  
diff --git a/target/linux/generic/backport-5.15/810-v5.17-net-qmi_wwan-add-ZTE-MF286D-modem-19d2-1485.patch b/target/linux/generic/backport-5.15/810-v5.17-net-qmi_wwan-add-ZTE-MF286D-modem-19d2-1485.patch
index 25802be7e3d03..41d3e121de195 100644
--- a/target/linux/generic/backport-5.15/810-v5.17-net-qmi_wwan-add-ZTE-MF286D-modem-19d2-1485.patch
+++ b/target/linux/generic/backport-5.15/810-v5.17-net-qmi_wwan-add-ZTE-MF286D-modem-19d2-1485.patch
@@ -49,7 +49,7 @@ Signed-off-by: David S. Miller <davem@davemloft.net>
 
 --- a/drivers/net/usb/qmi_wwan.c
 +++ b/drivers/net/usb/qmi_wwan.c
-@@ -1314,6 +1314,7 @@ static const struct usb_device_id produc
+@@ -1316,6 +1316,7 @@ static const struct usb_device_id produc
  	{QMI_FIXED_INTF(0x19d2, 0x1426, 2)},	/* ZTE MF91 */
  	{QMI_FIXED_INTF(0x19d2, 0x1428, 2)},	/* Telewell TW-LTE 4G v2 */
  	{QMI_FIXED_INTF(0x19d2, 0x1432, 3)},	/* ZTE ME3620 */
diff --git a/target/linux/generic/hack-5.15/251-kconfig.patch b/target/linux/generic/hack-5.15/251-kconfig.patch
index 0e24c3e8041f3..9350e366d8668 100644
--- a/target/linux/generic/hack-5.15/251-kconfig.patch
+++ b/target/linux/generic/hack-5.15/251-kconfig.patch
@@ -199,7 +199,7 @@ Signed-off-by: John Crispin <john@phrozen.org>
  	bool
 --- a/net/Kconfig
 +++ b/net/Kconfig
-@@ -432,7 +432,7 @@ config NET_DEVLINK
+@@ -430,7 +430,7 @@ config NET_DEVLINK
  	default n
  
  config PAGE_POOL
diff --git a/target/linux/generic/hack-5.15/280-rfkill-stubs.patch b/target/linux/generic/hack-5.15/280-rfkill-stubs.patch
index 7a650d132eb3a..ff6638f7a05b4 100644
--- a/target/linux/generic/hack-5.15/280-rfkill-stubs.patch
+++ b/target/linux/generic/hack-5.15/280-rfkill-stubs.patch
@@ -26,7 +26,7 @@ Signed-off-by: John Crispin <john@phrozen.org>
   * @name: name of the struct -- the string is not copied internally
 --- a/net/Makefile
 +++ b/net/Makefile
-@@ -52,7 +52,7 @@ obj-$(CONFIG_TIPC)		+= tipc/
+@@ -51,7 +51,7 @@ obj-$(CONFIG_TIPC)		+= tipc/
  obj-$(CONFIG_NETLABEL)		+= netlabel/
  obj-$(CONFIG_IUCV)		+= iucv/
  obj-$(CONFIG_SMC)		+= smc/
diff --git a/target/linux/generic/hack-5.15/721-net-add-packet-mangeling.patch b/target/linux/generic/hack-5.15/721-net-add-packet-mangeling.patch
index b4e92b4a185dc..69771cef398d9 100644
--- a/target/linux/generic/hack-5.15/721-net-add-packet-mangeling.patch
+++ b/target/linux/generic/hack-5.15/721-net-add-packet-mangeling.patch
@@ -38,7 +38,7 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
  
  /* Specifies the type of the struct net_device::ml_priv pointer */
  enum netdev_ml_priv_type {
-@@ -2015,6 +2020,7 @@ struct net_device {
+@@ -2014,6 +2019,7 @@ struct net_device {
  	/* Read-mostly cache-line for fast-path access */
  	unsigned int		flags;
  	unsigned int		priv_flags;
@@ -46,7 +46,7 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
  	const struct net_device_ops *netdev_ops;
  	int			ifindex;
  	unsigned short		gflags;
-@@ -2075,6 +2081,11 @@ struct net_device {
+@@ -2074,6 +2080,11 @@ struct net_device {
  	const struct tlsdev_ops *tlsdev_ops;
  #endif
  
@@ -58,7 +58,7 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
  	const struct header_ops *header_ops;
  
  	unsigned char		operstate;
-@@ -2149,6 +2160,10 @@ struct net_device {
+@@ -2145,6 +2156,10 @@ struct net_device {
  	struct mctp_dev __rcu	*mctp_ptr;
  #endif
  
diff --git a/target/linux/generic/hack-5.15/780-usb-net-MeigLink_modem_support.patch b/target/linux/generic/hack-5.15/780-usb-net-MeigLink_modem_support.patch
index 367d66fc666dc..96d800669a81e 100644
--- a/target/linux/generic/hack-5.15/780-usb-net-MeigLink_modem_support.patch
+++ b/target/linux/generic/hack-5.15/780-usb-net-MeigLink_modem_support.patch
@@ -29,7 +29,7 @@ Subject: [PATCH] net/usb/qmi_wwan: add MeigLink modem support
  
  #define QUECTEL_VENDOR_ID			0x2c7c
  /* These Quectel products use Quectel's vendor ID */
-@@ -1169,6 +1171,11 @@ static const struct usb_device_id option
+@@ -1173,6 +1175,11 @@ static const struct usb_device_id option
  	  .driver_info = ZLP },
  	{ USB_DEVICE(QUECTEL_VENDOR_ID, QUECTEL_PRODUCT_BG96),
  	  .driver_info = RSVD(4) },
diff --git a/target/linux/generic/pending-5.15/300-mips_expose_boot_raw.patch b/target/linux/generic/pending-5.15/300-mips_expose_boot_raw.patch
index 05d024925df0d..436c2cc767cff 100644
--- a/target/linux/generic/pending-5.15/300-mips_expose_boot_raw.patch
+++ b/target/linux/generic/pending-5.15/300-mips_expose_boot_raw.patch
@@ -9,7 +9,7 @@ Acked-by: Rob Landley <rob@landley.net>
 ---
 --- a/arch/mips/Kconfig
 +++ b/arch/mips/Kconfig
-@@ -1100,9 +1100,6 @@ config FW_ARC
+@@ -1101,9 +1101,6 @@ config FW_ARC
  config ARCH_MAY_HAVE_PC_FDC
  	bool
  
@@ -19,7 +19,7 @@ Acked-by: Rob Landley <rob@landley.net>
  config CEVT_BCM1480
  	bool
  
-@@ -3182,6 +3179,18 @@ choice
+@@ -3183,6 +3180,18 @@ choice
  		bool "Extend builtin kernel arguments with bootloader arguments"
  endchoice
  
diff --git a/target/linux/generic/pending-5.15/680-NET-skip-GRO-for-foreign-MAC-addresses.patch b/target/linux/generic/pending-5.15/680-NET-skip-GRO-for-foreign-MAC-addresses.patch
index 4ccf229026a28..f01348d349df7 100644
--- a/target/linux/generic/pending-5.15/680-NET-skip-GRO-for-foreign-MAC-addresses.patch
+++ b/target/linux/generic/pending-5.15/680-NET-skip-GRO-for-foreign-MAC-addresses.patch
@@ -11,7 +11,7 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
 
 --- a/include/linux/netdevice.h
 +++ b/include/linux/netdevice.h
-@@ -2101,6 +2101,8 @@ struct net_device {
+@@ -2100,6 +2100,8 @@ struct net_device {
  	struct netdev_hw_addr_list	mc;
  	struct netdev_hw_addr_list	dev_addrs;
  
diff --git a/target/linux/generic/pending-5.15/760-net-core-add-optional-threading-for-backlog-processi.patch b/target/linux/generic/pending-5.15/760-net-core-add-optional-threading-for-backlog-processi.patch
index d846a05a2487b..1ccdea6d993e1 100644
--- a/target/linux/generic/pending-5.15/760-net-core-add-optional-threading-for-backlog-processi.patch
+++ b/target/linux/generic/pending-5.15/760-net-core-add-optional-threading-for-backlog-processi.patch
@@ -20,7 +20,7 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
  
  /**
   *	napi_disable - prevent NAPI from scheduling
-@@ -3368,6 +3369,7 @@ struct softnet_data {
+@@ -3364,6 +3365,7 @@ struct softnet_data {
  	unsigned int		processed;
  	unsigned int		time_squeeze;
  	unsigned int		received_rps;
@@ -162,7 +162,7 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
  void netif_napi_add(struct net_device *dev, struct napi_struct *napi,
  		    int (*poll)(struct napi_struct *, int), int weight)
  {
-@@ -11371,6 +11446,9 @@ static int dev_cpu_dead(unsigned int old
+@@ -11369,6 +11444,9 @@ static int dev_cpu_dead(unsigned int old
  	raise_softirq_irqoff(NET_TX_SOFTIRQ);
  	local_irq_enable();
  
@@ -172,7 +172,7 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
  #ifdef CONFIG_RPS
  	remsd = oldsd->rps_ipi_list;
  	oldsd->rps_ipi_list = NULL;
-@@ -11710,6 +11788,7 @@ static int __init net_dev_init(void)
+@@ -11708,6 +11786,7 @@ static int __init net_dev_init(void)
  		sd->cpu = i;
  #endif
  
diff --git a/target/linux/lantiq/patches-5.15/0152-lantiq-VPE.patch b/target/linux/lantiq/patches-5.15/0152-lantiq-VPE.patch
index bf2962a3ffb7e..6776d35ecec37 100644
--- a/target/linux/lantiq/patches-5.15/0152-lantiq-VPE.patch
+++ b/target/linux/lantiq/patches-5.15/0152-lantiq-VPE.patch
@@ -15,7 +15,7 @@ Signed-off-by: Stefan Koch <stefan.koch10@gmail.com>
 
 --- a/arch/mips/Kconfig
 +++ b/arch/mips/Kconfig
-@@ -2429,6 +2429,12 @@ config MIPS_VPE_LOADER
+@@ -2430,6 +2430,12 @@ config MIPS_VPE_LOADER
  	  Includes a loader for loading an elf relocatable object
  	  onto another VPE and running it.
  
diff --git a/target/linux/layerscape/patches-5.15/701-staging-add-fsl_ppfe-driver.patch b/target/linux/layerscape/patches-5.15/701-staging-add-fsl_ppfe-driver.patch
index 6130bf0030ff3..e8a69153500f8 100644
--- a/target/linux/layerscape/patches-5.15/701-staging-add-fsl_ppfe-driver.patch
+++ b/target/linux/layerscape/patches-5.15/701-staging-add-fsl_ppfe-driver.patch
@@ -854,7 +854,7 @@ Signed-off-by: Pawel Dembicki <paweldembicki@gmail.com>
 +};
 --- a/MAINTAINERS
 +++ b/MAINTAINERS
-@@ -7526,6 +7526,14 @@ F:	drivers/ptp/ptp_qoriq.c
+@@ -7519,6 +7519,14 @@ F:	drivers/ptp/ptp_qoriq.c
  F:	drivers/ptp/ptp_qoriq_debugfs.c
  F:	include/linux/fsl/ptp_qoriq.h
  
diff --git a/target/linux/mediatek/patches-5.15/730-net-phy-add-driver-for-MediaTek-SoC-built-in-GE-PHYs.patch b/target/linux/mediatek/patches-5.15/730-net-phy-add-driver-for-MediaTek-SoC-built-in-GE-PHYs.patch
index 6af9e843357a0..43876deb66cbc 100644
--- a/target/linux/mediatek/patches-5.15/730-net-phy-add-driver-for-MediaTek-SoC-built-in-GE-PHYs.patch
+++ b/target/linux/mediatek/patches-5.15/730-net-phy-add-driver-for-MediaTek-SoC-built-in-GE-PHYs.patch
@@ -21,7 +21,7 @@ Signed-off-by: Daniel Golle <daniel@makrotopia.org>
 
 --- a/MAINTAINERS
 +++ b/MAINTAINERS
-@@ -11797,6 +11797,15 @@ S:	Maintained
+@@ -11790,6 +11790,15 @@ S:	Maintained
  F:	drivers/net/pcs/pcs-mtk-lynxi.c
  F:	include/linux/pcs/pcs-mtk-lynxi.h
  
diff --git a/target/linux/mvebu/patches-5.15/907-MAINTAINERS-Add-an-entry-for-the-IEI-WT61P803-PUZZLE.patch b/target/linux/mvebu/patches-5.15/907-MAINTAINERS-Add-an-entry-for-the-IEI-WT61P803-PUZZLE.patch
index b8accf180bb00..242a6c1e00abb 100644
--- a/target/linux/mvebu/patches-5.15/907-MAINTAINERS-Add-an-entry-for-the-IEI-WT61P803-PUZZLE.patch
+++ b/target/linux/mvebu/patches-5.15/907-MAINTAINERS-Add-an-entry-for-the-IEI-WT61P803-PUZZLE.patch
@@ -16,7 +16,7 @@ Cc: Robert Marko <robert.marko@sartura.hr>
 
 --- a/MAINTAINERS
 +++ b/MAINTAINERS
-@@ -9063,6 +9063,22 @@ F:	include/net/nl802154.h
+@@ -9056,6 +9056,22 @@ F:	include/net/nl802154.h
  F:	net/ieee802154/
  F:	net/mac802154/
  
diff --git a/target/linux/octeon/patches-5.15/100-mips_image_cmdline_hack.patch b/target/linux/octeon/patches-5.15/100-mips_image_cmdline_hack.patch
index 15e233aceb2c1..4b225277f9deb 100644
--- a/target/linux/octeon/patches-5.15/100-mips_image_cmdline_hack.patch
+++ b/target/linux/octeon/patches-5.15/100-mips_image_cmdline_hack.patch
@@ -10,7 +10,7 @@ Signed-off-by: Gabor Juhos <juhosg@openwrt.org>
 
 --- a/arch/mips/Kconfig
 +++ b/arch/mips/Kconfig
-@@ -1180,6 +1180,10 @@ config MIPS_MSC
+@@ -1181,6 +1181,10 @@ config MIPS_MSC
  config SYNC_R4K
  	bool
  
diff --git a/target/linux/ramips/patches-5.15/314-MIPS-add-bootargs-override-property.patch b/target/linux/ramips/patches-5.15/314-MIPS-add-bootargs-override-property.patch
index 07635a9d86a8d..ef45aac61bda9 100644
--- a/target/linux/ramips/patches-5.15/314-MIPS-add-bootargs-override-property.patch
+++ b/target/linux/ramips/patches-5.15/314-MIPS-add-bootargs-override-property.patch
@@ -17,7 +17,7 @@ Signed-off-by: David Bauer <mail@david-bauer.net>
 
 --- a/arch/mips/kernel/setup.c
 +++ b/arch/mips/kernel/setup.c
-@@ -544,8 +544,28 @@ static int __init bootcmdline_scan_chose
+@@ -545,8 +545,28 @@ static int __init bootcmdline_scan_chose
  
  #endif /* CONFIG_OF_EARLY_FLATTREE */
  
@@ -46,7 +46,7 @@ Signed-off-by: David Bauer <mail@david-bauer.net>
  	bool dt_bootargs = false;
  
  	/*
-@@ -559,6 +579,14 @@ static void __init bootcmdline_init(void
+@@ -560,6 +580,14 @@ static void __init bootcmdline_init(void
  	}
  
  	/*
diff --git a/target/linux/ramips/patches-5.15/315-owrt-hack-fix-mt7688-cache-issue.patch b/target/linux/ramips/patches-5.15/315-owrt-hack-fix-mt7688-cache-issue.patch
index f36f028cab384..d49429f7a3b86 100644
--- a/target/linux/ramips/patches-5.15/315-owrt-hack-fix-mt7688-cache-issue.patch
+++ b/target/linux/ramips/patches-5.15/315-owrt-hack-fix-mt7688-cache-issue.patch
@@ -10,7 +10,7 @@ Signed-off-by: John Crispin <blogic@openwrt.org>
 
 --- a/arch/mips/kernel/setup.c
 +++ b/arch/mips/kernel/setup.c
-@@ -686,7 +686,6 @@ static void __init arch_mem_init(char **
+@@ -687,7 +687,6 @@ static void __init arch_mem_init(char **
  	mips_reserve_vmcore();
  
  	mips_parse_crashkernel();
@@ -18,7 +18,7 @@ Signed-off-by: John Crispin <blogic@openwrt.org>
  
  	/*
  	 * In order to reduce the possibility of kernel panic when failed to
-@@ -803,6 +802,7 @@ void __init setup_arch(char **cmdline_p)
+@@ -804,6 +803,7 @@ void __init setup_arch(char **cmdline_p)
  
  	cpu_cache_init();
  	paging_init();
diff --git a/target/linux/realtek/patches-5.15/008-5.17-watchdog-add-realtek-otto-watchdog-timer.patch b/target/linux/realtek/patches-5.15/008-5.17-watchdog-add-realtek-otto-watchdog-timer.patch
index 17ff729f52565..dfe7ab28d73b9 100644
--- a/target/linux/realtek/patches-5.15/008-5.17-watchdog-add-realtek-otto-watchdog-timer.patch
+++ b/target/linux/realtek/patches-5.15/008-5.17-watchdog-add-realtek-otto-watchdog-timer.patch
@@ -32,7 +32,7 @@ Signed-off-by: Guenter Roeck <linux@roeck-us.net>
 
 --- a/MAINTAINERS
 +++ b/MAINTAINERS
-@@ -15902,6 +15902,13 @@ S:	Maintained
+@@ -15895,6 +15895,13 @@ S:	Maintained
  F:	include/sound/rt*.h
  F:	sound/soc/codecs/rt*
  
diff --git a/target/linux/realtek/patches-5.15/300-mips-add-rtl838x-platform.patch b/target/linux/realtek/patches-5.15/300-mips-add-rtl838x-platform.patch
index 6407a49fb6b62..f853ef4feef75 100644
--- a/target/linux/realtek/patches-5.15/300-mips-add-rtl838x-platform.patch
+++ b/target/linux/realtek/patches-5.15/300-mips-add-rtl838x-platform.patch
@@ -25,7 +25,7 @@ Submitted-by: Birger Koblitz <git@birger-koblitz.de>
  platform-$(CONFIG_SGI_IP28)		+= sgi-ip22/
 --- a/arch/mips/Kconfig
 +++ b/arch/mips/Kconfig
-@@ -1053,8 +1053,58 @@ config NLM_XLP_BOARD
+@@ -1054,8 +1054,58 @@ config NLM_XLP_BOARD
  	  This board is based on Netlogic XLP Processor.
  	  Say Y here if you have a XLP based board.
  
